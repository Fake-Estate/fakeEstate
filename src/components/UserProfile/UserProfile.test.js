import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import UserProfile from './UserProfile';
describe('AgentAuth', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<UserProfile debug />);
  
    expect(component).toMatchSnapshot();
  });
});