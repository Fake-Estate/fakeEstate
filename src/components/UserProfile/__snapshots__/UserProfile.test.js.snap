// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentAuth should render correctly in "debug" mode 1`] = `
<div>
  <h1>
    hi
  </h1>
</div>
`;
